vllm/entrypoints/openai/api_server.py

```diff
@@ -81,6 +117,16 @@ async def health() -> Response:
     return Response(status_code=200)
 
 
+if "--privileged" in sys.argv:
+
+    @app.get("/privileged")
```

Comment:
This being a GET (idempotent, can be retried by a user agent) and with a non-descriptive route path is a code smell. 

Should this be `/debug/reload-server`, and should it be a POST instead?

Reply:
Modified to --debug argument, and changed to @post(/debug/reload-server)